pkg/controller/kieapp/constants/constants.go

```diff
@@ -250,7 +250,7 @@ const (
 	ConsoleAuthoringMemLimit    = "4Gi"
 	ConsoleAuthoringCPURequests = "1500m"
 	ConsoleAuthoringMemRequests = "3Gi"
-	ConsoleProdCPURequests      = "500m"
+	ConsoleProdCPURequests      = "1500m"
```

Comment:
just one more thing, ConsoleProdCPULimit is less than ConsoleProdCPURequests.
please set the ConsoleProdCPULimit to 2.

Reply:
Thanks for picking up on that, updated the same.